src/sentry/runner/commands/cleanup.py

```diff
@@ -198,12 +199,11 @@ def _cleanup(
         try:
             from sentry.runner import configure
 
-            configure()
+            if not settings.configured:
```

Comment:
This is necessary for adding tests since calling `configure()` within the context of pytest conflicts. I intend to add tests on a later PR.

Reply:
Probably we should call `configure()` in the click method instead of having a branch here. Probably `_validate_and_setup_environment` can move there too. Then you can call this directly from tests